.circleci/config.yml

```diff
@@ -177,9 +229,76 @@ jobs:
           path: /tmp/resource-usage
           destination: resource-usage
 
+  unit-test-windows:
+    executor:
+      name: win/default
+      shell: bash.exe
+    steps:
+      - dependencies-windows
+
+      # - run:
```

Comment:
Should either port this to Windows or delete it. Probably fine to delete it. (Useful auxiliary debugging information if test go awry, but we can add it if/when needed.)